src/azure-cli/HISTORY.rst

```diff
@@ -3,6 +3,132 @@
 Release History
 ===============
 
+2.34.0
+++++++
+
+**ACR**
+
+* `az acr manifest`: Add new command group to support managing artifact manifests in Azure Container Registries (#21161)
+* Deprecate `az acr repository show-manifests` command and replace with `acr manifest metadata list` command (#21161)
+
+**AKS**
+
+* `az aks nodepool update`: Add `--node-taints` to allow modify node taints (#21138)
+* `az aks get-credentials`: Add new parameter `--format` to support specifying the format of returned credential (#21383)
+* `az aks nodepool`: Allow specifying `--scale-down-mode` in nodepool create and update (#21396)
+
+**API Management**
+
+* apim api import: Update api-id description  #18306 (#21248)
+
+**APIM**
+
+* Fix #21187: `az apim api create/update/import`: Fix header and query param names being swapped (#21202)
+
+**App Config**
+
+* `az appconfig kv import`: Add new parameter `--strict` to support strict import. (#21067)
+
+**App Service**
+
+* [BREAKING CHANGE] `az webapp up` : Change supported runtimes (#21057)
+* [BREAKING CHANGE] `az webapp create` : Change supported runtimes (#21057)
+* [BREAKING CHANGE] `az webapp list-runtimes` : Add `--os`/`--os-type` argument, change runtimes, change default behavior to return both linux and windows stacks, and deprecate `--linux` argument (#21057)
+* [BREAKING CHANGE] `az functionapp create` : Take runtime names and versions from API instead of hardcoded list (#21057)
+* `az functionapp plan`: Update the max value of `--max-burst` to 100 (#21233)
+* `az functionapp list-runtimes` : Add new command to show function app runtimes, versions, and compatible functions versions (#21057)
```

Comment:
```suggestion
* [BREAKING CHANGE] `az webapp up`: Change supported runtimes (#21057)
* [BREAKING CHANGE] `az webapp create`: Change supported runtimes (#21057)
* [BREAKING CHANGE] `az webapp list-runtimes`: Add `--os`/`--os-type` argument, change runtimes, change default behavior to return both linux and windows stacks, and deprecate `--linux` argument (#21057)
* [BREAKING CHANGE] `az functionapp create`: Take runtime names and versions from API instead of hardcoded list (#21057)
* `az functionapp plan`: Update the max value of `--max-burst` to 100 (#21233)
* `az functionapp list-runtimes`: Add new command to show function app runtimes, versions, and compatible functions versions (#21057)
```